getter/utils.go

```diff
@@ -0,0 +1,100 @@
+package getter
+
+import (
+	"bytes"
+	"compress/gzip"
+	_ "embed"
+	"encoding/csv"
+	"io"
+	"strconv"
+	"sync"
+
+	locodedb "github.com/nspcc-dev/locode-db/cmd/locode/db"
+)
+
+//go:embed data/countries.csv.gz
+var countriesData []byte
+
+//go:embed data/locodes.csv.gz
+var locodesData []byte
+
+var locodeDataOnce sync.Once
+
+func InitLocodeData() {
```

Comment:
Hide it. Call it from `Get`. The first access will be slow, but subsequent ones will work fine.